content/en/docs/test.md

```diff
@@ -92,10 +92,10 @@ culpa qui officia deserunt mollit anim id est laborum.
 3.  For single-digit numbered lists, using two spaces after the period makes
     interior block-level content line up better along tab-stops.
 
+    <!-- separate lists -->
```

Comment:
@bep any idea why the numbered lists are not divided, even with this comment here?

Reply:
OK apparently this was because it has to be on the left-hand margin. SO GROSS.